src/cpp/dds/publisher/qos/PublisherQos.cpp

```diff
@@ -0,0 +1,205 @@
+// Copyright 2019 Proyectos y Sistemas de Mantenimiento SL (eProsima).
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @file PublisherQos.cpp
+ *
+ */
+
+#include <fastdds/dds/publisher/qos/PublisherQos.hpp>
+#include <fastrtps/log/Log.h>
+
+using namespace eprosima::fastdds::dds;
+
+const PublisherQos eprosima::fastdds::dds::PUBLISHER_QOS_DEFAULT;
+
+PublisherQos::PublisherQos()
+{
+    this->reliability.kind = fastrtps::RELIABLE_RELIABILITY_QOS;
+    this->durability.kind = fastrtps::TRANSIENT_LOCAL_DURABILITY_QOS;
+}
+
+PublisherQos::~PublisherQos()
+{
+
+}
+
+
+void PublisherQos::set_qos(const PublisherQos& qos, bool first_time)
+{
+    if(first_time)
```

Comment:
Coding style (Review all lines)
```suggestion
    if (first_time)
```

Reply:
I moved this file, but I did the change...